extensions/interactions/LogicProof/directives/LogicProof.js

```diff
@@ -86,7 +86,7 @@ oppia.directive('oppiaInteractiveLogicProof', [
             $scope.questionData.language.operators);
           $scope.questionString = (
             $scope.assumptionsString === '' ?
-              'I18N_INTERACTIONS_LOGIC_PROOF_QUESTION_STR_NO_ASSUMPTION':
+              'I18N_INTERACTIONS_LOGIC_PROOF_QUESTION_STR_NO_ASSUMPTION' :
```

Comment:
Why this change?

Reply:
Hi @kashifsoofi really sorry for the delay! I completely missed this ping! 

There isn't a space on develop see [here](https://github.com/oppia/oppia/blob/develop/extensions/interactions/LogicProof/directives/LogicProof.js#L89). Could you revert this change? And let's see if the lint checks continue to pass.

Reply:
@nalinbhardwaj done